experiments/generic/disk_loss/disk_loss_ansible_logic.yml

```diff
@@ -47,10 +48,20 @@
         - name: Gcloud authentication
           include_tasks: "/utils/cloud/gcp/gcloud_configure.yml"
           when: "cloud_platform == 'GCP'"
+        
+        # AWS authentication
+        - name: AWS authentication
+          include_tasks: "/utils/cloud/aws/aws_configure.yml"
+          when: "cloud_platform == 'AWS'"
 
         ## PRE-CHAOS DISK LIVENESS CHECK
         - name: Verify that the disk is connected to node (pre)
           include_tasks: "/utils/cloud/gcp/status_disk.yml"
+          when: "cloud_platform == 'GCP'"
+
+        - name: Verify that the disk is connected to node (pre)
+          include_tasks: "/utils/cloud/aws/status_disk.yml"
+          when: "cloud_platform == 'AWS'"
 
         # Checking disk is attached to node
         - debug: 
```

Comment:
what debug?

Reply:
As @ksatchit . He told to use debug module